### PR TITLE
tags -- starting new branch bc of terrible rebase issues

### PR DESCRIPTION
**Pull request by mcgingras**
Opening a new branch to fix rebase issues. I was having trouble rebasing so I wanted to make a new branch so we could have a cleaner git history.

### CODE REVIEW

File: packages/nouns-webapp/tsconfig.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "jsx": "react-jsx"
   },
   "include": [
-    "src"
+    // "src"
```
Comment (mcgingras) [inline packages/nouns-webapp/tsconfig.json:17]: reminder to uncomment this  -- unfortunately its the only way I'm able to run the project locally without fixing like 100 typescript warnings .

**Comment by mcgingras** [comment]
I rebased from master but this PR is pointed at `filter-schema-updates` so I think we need to rebase that one as well.

**Comment by mcgingras** [comment]
> On the point about tags like Hot I'd probably create a map for these "virtual tags" and then load them in here. Something like this?
> 
> ```
> // load + build DB tags
> 
> const virtualTags = {
>   HOT: {
>     type: HOT,
>     displayName: "Hot",
>     filterFn: (ideas) => { ... logic to filter ideas }, // not sure about this property
>   },
>   ...
> }; // Maybe store these in a standalone file and import them here - virtualTags.ts?
> 
>  const virtualFilterOptions = Object.keys(virtualTags).map(vT => {
>   return {
>      id: `${FILTER_IDS.TAG}-${virtualTags[vT].type}`
>      label: tag.displayName,
>      value: buildFilterParam(FILTER_IDS.TAG, virtualTags[vT].type),
>      selected: Boolean(
>          root.tagParams?.includes(buildFilterParam(FILTER_IDS.TAG, virtualTags[vT].type)),
>      ),
>   }
> })
> 
> 
> const tagFilter: PropLotFilter = {
>  ....
>  options: [...filterOptions, ...virtualFilterOptions],
> }
> ```

**Comment by ThatTobMate** [comment]
As i'd merged a few different PRs and caused more conflicts on yours i've rebased this one for you [here](https://github.com/ThatTobMate/lilnouns-monorepo/pull/41) :)

**Pull request closed without merging**